indico/htdocs/js/indico/modules/designer.js

```diff
@@ -618,17 +633,59 @@
         $('.backside-tools').toggleClass('hidden', showPlaceholder);
     }
 
-    function toggleWarningOnDimensionChange(xDimension, yDimension, initialXDimention, initialYDimention) {
-        var $backsideTplsWarning = $('.affected-targets-warning');
-        if (xDimension.val() * DEFAULT_PIXEL_CM !== initialXDimention) {
-            $backsideTplsWarning.show('fast');
-            showConfirmationDialog = true;
-        } else if (yDimension.val() * DEFAULT_PIXEL_CM === initialYDimention) {
-            $backsideTplsWarning.hide('fast');
-            showConfirmationDialog = false;
+    function togglePlaceholderToBeRemoved(showPlaceholder) {
+        toggleBacksidePlaceholder(showPlaceholder);
+        $('.backside-tools').toggleClass('hidden', !showPlaceholder);
+        if (showPlaceholder) {
+            $('.back-side-placeholder').addClass('to-be-removed');
+            $('.placeholder-text').html($T.gettext('This back side template will be removed'));
+            $('.placeholder-link').hide();
+            confirmMessage = $T.gettext('The back side of this template will be removed');
+            confirmTitle = $T.gettext('Disassociate back side');
+            removeBackSide = true;
+        } else {
+            $('.back-side-placeholder').removeClass('to-be-removed');
+            $('.placeholder-text').html($T.gettext('The back side is empty.'));
+            $('.placeholder-link').show();
+            confirmMessage = $T.gettext("The back side of the templates that use this one as the back side will be removed.");
+            confirmTitle = $T.gettext("Disassociate back sides");
+            removeBackSide = false;
+        }
+    }
+
+    function toggleWarningOnDimensionChange(xDimension, yDimension, initialXDimension, initialYDimension) {
+        if ($('#back-side-warning').length || backsideTemplateID !== null) {
```

Comment:
why not negate the condition and return early instead of wrapping the whole function body inside the if?